### DISCUSSION TITLE
Change order of plugin injected elements in entry side-bar

### DISCUSSION THREAD

**Discussion by WillemMoeskops** [craftcms/cms#14845] [Ideas]
Plugins have the ability to inject custom code in details-container in right sidebar of an entry. It now gets injected based on alphabetic order of the plugin name. It would be great to be able to change the order of this.

**Comment by brandonkelly** [comment]
`EVENT_DEFINE_SIDEBAR_HTML` handlers are just modifying the `DefineHtmlEvent::$html` property, so they could choose to prepend their sidebar contents rather than append them.

If we were to make it customizable, that would probably be more of a field layout designer thing…

**Reply by WillemMoeskops** [reply]
That sounds good to me! 😁
